spec/ruby/core/io/each_byte_spec.rb

```diff
@@ -15,6 +15,16 @@
     -> { IOSpecs.closed_io.each_byte {} }.should raise_error(IOError)
   end
 
+  ruby_version_is '3.0' do
```

Comment:
This should be `ruby_bug '#XXXXX', ''...'3.0.1' do`, where XXXXX is the issue number at bugs.ruby-lang.org.

Reply:
There isn't an issue number because my account on bugs.ruby-lang.org is broken (I tried to reset the password some months ago, but the password reset appears broken, and now I can not log in at all). If I could log in, you'd have that issue number, but if someone else wants to create it, I'll happily reference that number re the advice above.

Thank you Nobu.

Reply:
I opened a ticket [here](https://bugs.ruby-lang.org/issues/17661). It's `[Bug #17661]`.

Reply:
Thank you @tenderlove; I will get the PR updated accordingly.